recipe-server/normandy/control/templates/control/new.html

```diff
@@ -4,7 +4,7 @@
 <!DOCTYPE html>
 {% get_current_language as LANGUAGE_CODE %}
 
-<html lang="{{ LANGUAGE_CODE|default:"en-us" }}">
+<html lang="{{ LANGUAGE_CODE|default:"en-us" }}" data-csrf="{{csrf_token}}">
```

Comment:
This should be safe for removal, right? As far as I can tell, the new `service_info` API endpoint should/would return the csrf token if needed, beyond that it looks like `apiFetch` pulls the actual cookie value itself.